jx_sqlite/query_table.py

```diff
@@ -229,12 +229,16 @@ def query(self, query):
                     allowNulls=allowNulls,
                     domain=domain
                 ))
+                
+            zeros = []
+            data_cubes = {}
+            for si, s in enumerate(listwrap(query.select)):
+                if s.aggregate == "count" and index_to_columns[si].push_child == ".":
+                    zeros.append(0)
```

Comment:
the old version had `len(zeros)==len(listwrap(query.select))`; nothing is appended to `zeros` in the `else` block to maintain that identity.